packages/f-searchbox/src/components/formElements/FormSearchField.vue

```diff
@@ -22,42 +22,115 @@
                     [$style['c-search-input']]: true,
                     [$style['is-notEmpty']]: address
                 }"
-                @input="$emit('input', $event.target.value)">
+                @input="$emit('input', $event.target.value)"
+                @focus="toggleEnterLeaveInput(true)"
+                @blur="toggleEnterLeaveInput(false)">
 
             <span :class="$style['c-search-placeholder']">{{ copy.fieldPlaceholder }}</span>
         </label>
+
+        <form-search-inner-field-wrapper
+            v-model="streetNumber"
+            :copy="copy"
+            :service="service" />
     </div>
 </template>
 
 <script>
+import { mapActions, mapState } from 'vuex';
+import FormSearchInnerFieldWrapper from './FormSearchInnerFieldWrapper.vue';
+
+const ALLOWED_SELECTION_TIME = 500;
+
 export default {
+    components: {
+        FormSearchInnerFieldWrapper
+    },
     props: {
         errorMessage: {
             type: [Boolean, String],
             default: false
         },
+
         address: {
             type: String,
             default: ''
         },
+
         isCompressed: {
             type: Boolean,
             default: true
         },
+
         copy: {
             type: Object,
             default:  () => ({})
         },
+
         shouldDisplayCustomAutocomplete: {
             type: Boolean,
             default: false
+        },
+
+        streetNumber: {
+            type: String,
+            default: ''
+        },
+
+        service: {
+            type: Object,
+            default: () => {}
         }
     },
 
     computed: {
+        ...mapState('searchbox', [
+            'shouldInputFieldHaveFocus'
+        ]),
+
         getAddressValue () {
             return this.address;
         }
+    },
+
+    watch: {
+        /**
+         * We want to make sure that we focus the user on the input field when
+         * results are returned. The `shouldInputFieldHaveFocus` is set via the
+         * mutation in the store. We watch for a change after a successful response
+         * and only then do we want to focus.
+         *
+         * */
+        shouldInputFieldHaveFocus (value) {
+            if (value) {
+                this.$refs.addressInput.focus();
+            }
+        }
+    },
+
+    methods: {
+        ...mapActions('searchbox', [
+            'setInputFocus'
+        ]),
+
+        /**
+         * Set input focus value (boolean) when the user enters & leaves
+         * the input field. We need to delay the blur event so users can make a selection,
+         * if we don't delay it the suggestions disappear before they can click on their choice.
+         *
+         * DELAY: `ALLOWED_SELECTION_TIME: 500ms`
+         *
+         * @param value
```

Comment:
It's useful to add type hints to help with IDE autocompletion - and for a human quickly scanning to see what a function accepts e.g.
```suggestion
         * @param {Boolean} value
```

Reply:
Fixed.